assets/src/design-system/theme/helpers/scrollbar.js

```diff
@@ -13,30 +13,36 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 /**
  * External dependencies
  */
 import { css } from 'styled-components';
-import { rgba } from 'polished';
 
-export const ScrollBarStyles = css`
+/*
+ * CSS for custom gray scrollbars for Chromium, Safari, Edge, & Opera.
+ *
+ * NOTE: Firefox does not yet support webkit scrollbar styling
+ */
+export const scrollbarCSS = () => css`
+  /* Firefox compatible css */
+  scrollbar-width: thin;
+  scrollbar-color: ${({ theme }) => theme.colors.interactiveBg.secondaryNormal};
+  /* end Firefox compatible css */
+
   ::-webkit-scrollbar {
-    width: 2px;
+    width: 11px;
+    height: 11px;
   }
 
   ::-webkit-scrollbar-track {
-    background-color: transparent;
+    background: ${({ theme }) => theme.colors.inverted.divider.secondary};
   }
 
   ::-webkit-scrollbar-thumb {
-    border: none;
+    background-clip: content-box;
     background-color: ${({ theme }) =>
-      rgba(theme.DEPRECATED_THEME.colors.bg.white, 0.38)};
-  }
-
-  :focus {
-    outline: 2px solid
-      ${({ theme }) => theme.DEPRECATED_THEME.colors.accent.secondary};
+      theme.colors.interactiveBg.secondaryNormal};
+    border: 2px solid transparent;
```

Comment:
In the figma designs there is a box-shadow here. I had trouble adding it in since the border is what I'm using to shrink the thumb of the scrollbar.

If anybody has any ideas, that would be greatly appreciated!

Reply:
I don't feel strongly about this. I'm okay with it as is. Also, I don't see a clean way of doing this (maybe not even any at all).

UX needs to approve of course. Please post in the design slack for feedback from UX team.